README.md

```diff
@@ -1 +1,78 @@
-# encorpora.io
+# Corpora Inc - Official Website
```

Comment:
I actually own `corpora.inc` domain as well. I'm thinking `encorpora` is things that we have build "on corpora" .. Corpora Inc is "any legal prupose" so it could be broader over time. Encorpora is the apps and books that we build with the core corpora software ...